Update azure-security-disk-encryption.md

mabicca opened the pull request:
Adding notes about disabling data drive encryption for Linux if the OS drive is encrypted. That is currently not supported.

azurecla commented:
Hi __@oranms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (oranbr). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

mabicca committed:
Update azure-security-disk-encryption.md

Adding notes about disabling data drive encryption for Linux if the OS drive is encrypted. That is currently not supported.
